src/main/java/org/infinity/UploadServlet.java

```diff
@@ -67,7 +67,7 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws S
                 document.setSize(filePart.getSize());
                 document.setSha1(sb.toString());
                 DataManagement.getInstance().insertFile(document);
-            }else {
+            } else {
```

Comment:
It is very important. If you can lose only one whitespace then you can lose all your life!